code/obj/critter/sword.dm

```diff
@@ -1,4 +1,5 @@
 #define SWORD_ATTACKING_RANGE 4
+#define IS_ARRIVALS(x) (istype(x, /area/shuttle/arrival/station) || istype(x, /area/station/crewquarters/cryotron) || istype(x, /area/station/hallway/secondary/oshan_arrivals))
```

Comment:
Might be good to put in a more general place? At the least it'll probably come in useful when someone gets around to fixing emagged bots getting into arrivals.

Reply:
if a dev thinks it'll be useful and is Good Enough™ I could shove it into `_std/macros/area.dm` probably

Reply:
Is a decent idea.